docs/12_glossary.adoc

```diff
@@ -51,6 +51,8 @@ Potentially more columns in case you need translations.
 | mui | Npm library used to optimize the development of GUI. It gives us a lot of improvements while creating buttons, panels and many other things. The link to the webpage can be found link:https://mui.com/[here].
 | mongodb | Official mongodb driver for Node.js. Usage documentation can be found in this link:https://www.npmjs.com/package/mongodb[webpage].
 |mongoose| Object modeling tool designed to work in an asynchronous environment. Mongoose supports both promises and callbacks. Here is the link to the link:https://www.npmjs.com/package/mongoose[documentation].
-|Picker express API| Api which helps to calculate orders' price and many other things related to order's management. Here is the link:https://www.pickerexpress.com/[docuementation].
+|inrupt/solid-ui-react| Module which improves the development of applications using react and Solid architecture. The link to th documentation is link:https://www.npmjs.com/package/@inrupt/solid-ui-react[here].
+|node-fetch| Module which is useful to make requests to other web services. Here is the link to the link:https://www.npmjs.com/package/node-fetch[documentation].
+|MapsAPI| Api which helps to calculate distances between the client and the distribution centre. We will use Mpabox in order to implement this features. Here is the link:https://docs.mapbox.com/api/overview/[documentation].
```

Comment:
center en vez de centre
Mapbox en vez de Mpabox
these features en vez de this features